compiler/extccomp.nim

```diff
@@ -672,6 +672,12 @@ proc callCCompiler*(projectfile: string) =
   var prettyCmds: TStringSeq = @[]
   let prettyCb = proc (idx: int) =
     echo prettyCmds[idx]
+  let runCb = proc (idx: int, p: Process) =
+    let exitCode = p.peekExitCode
+    if exitCode != 0:
+      echo p.outputStream.readAll
```

Comment:
You must not use 'echo' here. Every output needs to go through msgs.nim which knows about nimsuggest (Nim running as a server) etc.